src/index.html

```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Colored blocks</h1>
+  <div id="b01"></div>
```

Comment:
use class names that describe the content or behavior of the data, abstract names are a bad practice

Reply:
ids are not recommended for styling elements, they have high specificity and they can't be reused,
use class names instead